addons/website_slides/models/slide_channel.py

```diff
@@ -157,15 +157,13 @@ def unlink(self):
         Override unlink method :
         Remove attendee from a channel, then also remove slide.slide.partner related to.
         """
-        removed_slide_partner_domain = []
-        for channel_partner in self:
-            # find all slide link to the channel and the partner
-            removed_slide_partner_domain = expression.OR([
-                removed_slide_partner_domain,
-                [('partner_id', '=', channel_partner.partner_id.id),
-                 ('slide_id', 'in', channel_partner.channel_id.slide_ids.ids)]
-            ])
-        if removed_slide_partner_domain:
+        # find all slide link to the channel and the partner
+        removed_slide_partner_domain = expression.OR([
+            [('partner_id', '=', channel_partner.partner_id.id),
+                ('slide_id', 'in', channel_partner.channel_id.slide_ids.ids)]
```

Comment:
Just nitpicking about identation :wink: 
```suggestion
            [('partner_id', '=', channel_partner.partner_id.id),
             ('slide_id', 'in', channel_partner.channel_id.slide_ids.ids)]
```